ckanext/search/tests/test_filters.py

```diff
@@ -317,12 +333,69 @@ def test_filters_list_of_filters_with_or_operator():
     )
 
 
+def test_filters_unknown_field_single_field(default_search_schema):
+    filters = {"random_field": {"eq": "value"}}
+    with pytest.raises(ValidationError) as e:
+        query_filters_validator(filters, default_search_schema)
+
+    assert e.value.error_dict["filters"][0] == "Unknown field: random_field"
```

Comment:
I tend to write these like this so failures don't turn into errors
```suggestion
    assert e.value.error_dict == {'filters':["Unknown field: random_field"]}
```